### PR TITLE
Update People

### PR DESCRIPTION
**Pull request by AmberLJC**

### CODE REVIEW

**Review by jaywonchung — COMMENTED** [review]
I think we should stick to unifying numbers in ordered lists to all 1. Especially for current member sections (Ph.D. and non-Ph.D.), people come and go in random orders and it's annoying to sort the numbers again every time. On the other hand, if everything is just 1, we can just delete people cleanly. Note that HTML rendering is identical regardless of the numbers.

BTW, isn't the entry for Haochen Zhang strange? Did the person leave or not?

**Comment by AmberLJC** [comment]
Updated! 
Haochen is still with us but graduated.

**Comment by jaywonchung** [comment]
How did we do it for Yinwei after he got accepted to Princeton? Didn't he just remain in the Current Members section without the arrow until he was moved to the Past Members section?

**Comment by jaywonchung** [comment]
I'll delegate that decision to @mosharaf. At the minimum, I think the arrow should be that single-character arrow in the Past Members section, not `-->`.

**Comment by AmberLJC** [comment]
Yea, agreed

**Comment by jaywonchung** [comment]
Yeah it's just I feel like if a person is in SymbioticLab, they're in SymbioticLab. They may have a fixed future career step, but that future didn't come yet XD

File: source/people/index.md [hunk 1]
```diff
@@ -58,3 +63,4 @@ date: 2022-05-05 10:44:00
 28. Xiang Shen (CSE BSE) → Meta
 29. Churong Ji (CSE BSE) → Master's @CMU
 30. Sanjay S. Singapuram (CSE Ph.D.) → Ph.D. @Michigan
+31. Hongxi Pu (CSE BSE) → Master’s @Michigan
```
Comment (mosharaf) [inline source/people/index.md:66]: Wrong apostrophe. Should be '

File: source/people/index.md [hunk 2]
```diff
@@ -25,7 +25,12 @@ date: 2022-05-05 10:44:00
 1. Yuxuan Zhu (CSE BSE)
 1. Yutong Li (Math & CS-LSA BS)
 1. Zayd Uzzaman (Greenhills School)
-1. Luoxi Meng (CSE MSE)
+1. Yile Gu (CSE MSE) 
+1. Andrew Li (CSE BSE)
+1. Haochen Zhang (CSE BSE) -->  Master’s@Rice
```
Comment (mosharaf) [inline source/people/index.md:30]: I think we can (1) either move him to past member, or (2) use (CSE BSE → Master's @Rice) as affiliation. I prefer the former unless he's actively working on something. 

Also, wrong apostrophe. Should be '

And a missing space before @Rice.

**Review by mosharaf — APPROVED** [review]
LGTM

**Pull request merged**